magento/magento2
# Feature australian regions

Add Australian regions to Magento Vanila Install

### Description
Add Australian regions to be as a dropdown during address population

### Fixed Issues (if relevant)
1. magento/magento2#17514: Add Australian regions 

### Manual testing scenarios
1. Add product to cart
2. Go to checkout
3. Select Australia as shipping / billing country
4. I see Australian regions as a dropdown 

### Contribution checklist
